src/components/Transaction.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react'
 
-const Transaction = () => {
+const Transaction = (props) => {
   return (
-    <tr>
-      <td>{"...your code here"}</td>
-      <td>{"...your code here"}</td>
-      <td>{"...your code here"}</td>
-      <td>{"...your code here"}</td>
+    <tr >
+      <td>{props.details.posted_at}</td>
```

Comment:
I would name this `transaction` instead of `details`. I'd also use de-structuring to pull that object off of props to make it more readable.

`const {transaction} = props`